robottelo/config/validators.py

```diff
@@ -255,6 +255,7 @@
             'repos.rhscl_repo',
             'repos.ansible_repo',
             'repos.swid_tools_repo',
+            'repos.ohsnap_host',
```

Comment:
Its still missing in conf template file and also you need to open MR in CI for adding this setting in conf file.

Reply:
CI PR is already merged `649 `